src/backend/InvenTree/order/serializers.py

```diff
@@ -967,6 +975,7 @@ def save(self):
                         status=item['status'],
                         barcode=item.get('barcode', ''),
                         batch_code=item.get('batch_code', ''),
+                        expiry_date=item.get('expiry_date', ''),
```

Comment:
Change this to `expiry_date=item.get('expiry_date'),

Reply:
For this particular one, I ended up using `.get('expiry_date', None)` as it matched the pattern of those around it, even though the `None` is technically redundant.